src/services/emptyDocuments.ts

```diff
@@ -93,6 +93,59 @@ const emptyDocuments: Document[] = [
             },
         ],
     } as Document,
+    {
+        type: "isaacBookIndexPage",
+        encoding: "markdown",
+        title: "New Book Index Page",
+        coverImage: {
+            src: "",
+            type: "image",
+            altText: ""
+        },
+        children: [
+            {
+                type: "content",
+                encoding: "markdown",
+                value: "Add page content here",
+            } as Content,
+        ],
+        sidebar: "",
```

Comment:
Hmmn. This line (setting to the empty string) and [the code that uses the sidebar property](https://github.com/isaacphysics/isaac-api/blob/master/src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/ContentMapper.java#L248) won't play together nicely. I think this is a bug in the Java, though, and should be fixed there. I'll do that.